packages/web/src/public/locales/ar/translation.json

```diff
@@ -1,47 +1,28 @@
 {
-  "language_one": "لغة",
+  "language_one": "اللغة",
```

Comment:
Why so many changes in this file?

Reply:
I didn't want to reorganize the file, so I just asked chatgpt to translate the entire file. It apparently came up with something different than google translate for a few of the keys